amgut/templates/emperor.html

```diff
@@ -0,0 +1,288 @@
+          switch(cat) {
+            case 'age':
+              cat = 'age_cat';
+              break;
+            case 'bmi':
+              cat = 'bmi_cat';
+              break;
+            case 'sex':
+              cat = 'sex';
+              break;
+            default:
+            cat = 'all';
+          }
+          ec.colorController.$select.val(cat);
+          ec.colorController.$select.trigger("change");
+        });
+
+
+        //HIDE EDITING USING EMPEROR
+        $(function(){
+          //var barcodes = getParameterByName('barcodes').split(',');
+          ec.colorController.$header.hide();
+          ec._$tabsList.hide();
+          ec.colorController.$select.val('all');
+          ec.colorController.$select.trigger("change");
+
+          //CHANGE BG TO WHITE AND AXES TO BLACK
+          sv.drawAxesLabelsWithColor('#000000');
+          sv.drawAxesWithColor('#000000');
+          sv.renderer.setClearColor('#FFFFFF');
+
+          rescale();
+
+        //RECOLOR TO HIGHLIGHT POINTS
+          s = sv.decViews.scatter;
+          s.setGroupColor(0xFF0000, s.decomp.getPlottableByIDs([$("#barcode-fold").val()]));
+        });
+      });
+  }); // END REQUIRE.JS block
+
+  $(document).ready(function () {
+    var ctf = document.getElementById("canvas-bar-fold").getContext("2d");
+    barChartSummaryData.datasets = collapse(barChartSummaryData.metaData, 'phylum', 10)
+    window.foldBar = new Chart(ctf, {
+        type: 'bar',
+        data: barChartFoldData,
+        barStrokeWidth:0,
+        options: {
+            legend: {
+              display: false
+            },
+            title:{
+                display:true,
+                text:"Taxonomic Fold Changes"
+            },
+            tooltips: {
+                mode: 'single',
+            },
+            responsive: true,
+            scales: {
+                xAxes: [{
+                  ticks: {
+                    autoSkip: false
+                  }
+                }],
+                yAxes: [{
+                    display: true,
+                    scaleLabel: {
+                      display: true,
+                      labelString: 'Log2 Fold Change'
+                    }
+                }]
+            }
+        }
+    });
+    fold_change();
+    $("#collapse-fold").on("change", fold_change);
+  });
+</script>
+{% end %}
+{% block content %}
+<div id="taxa-fold-wrapper" style="width:100%; position: relative;">
+    <div id="taxa-fold-controls">
+    <table style="width: 70%"><tr><td>
+      <h3>Explore a sample in detail</h3>
+      Sample:
+      <select id="barcode-fold">
+    {% for bc in barcodes %}
+        <option value="{{bc['barcode']}}">{{bc['barcode']}} - {{bc['sample_date'].strftime('%b %d, %Y')}}</option>
+    {% end %}
+      </select>
+    </td><td>
+    <h3>Compare with population averages</h3>
+    Select taxa level <a href="" onclick="return false;" title="THIS IS THE TAXA TOOLTIP">?</a>:
```

Comment:
There is still placeholder text everywhere, which will get replaced in another PR. This is going into another branch so we can iterate on the design and text on the AG test server. This PR is concerned with getting the functionality in.